stacks/apps/augury.yml

```diff
@@ -631,6 +631,8 @@ Resources:
               Value: "true"
             - Name: POSTGRES_POOL_SIZE
               Value: !Ref kPostgresPoolSize
+            - Name: POSTGRES_STATEMENT_TIMEOUT
+              Value: 60000
```

Comment:
My guess is that this value isn't in seconds, so I'd maybe add a comment indication the scale or units